SayTheirNamesTests/DateFormatterServiceTest/DateFormatterServiceTests.swift

```diff
@@ -14,15 +14,21 @@ class DateFormatterServiceTests: XCTestCase {
     let dateFormatterService = DateFormatterService()
     
     func testDateOfBirthFormat() {
-        let dob = Date.dateFrom(year: 2020, month: 06, day: 03)!
```

Comment:
here `!` can simply be removed - `XCTAssertEqual` handles nil and throwing out of the box, by design

Reply:
@hybridcattt yeah but the thing is the dob isn't put into `XCAssertEqual`. The func that is called does not accept the nil

Reply:
riiiiight, that's not the thing being tested. cool then 👍